tests/processor.js

```diff
@@ -0,0 +1,48 @@
+const faker = require('faker');
+
+module.exports = {
+
+  generateProductId: (context, ee, next) => {
+    context.vars.productId = Math.floor(Math.random() * 1000000) + 9000000;
+    return next();
```

Comment:
I see you found a nuch cleaner way of introducing the random productIds. Looks good